lib/tccMessage.js

```diff
@@ -242,20 +242,20 @@ function stateValidValues(thermostat) {
 }
 
 function targetState(thermostat) {
-  // TCC to HomeKit
-  var state;
```

Comment:
The variable state needs to be defined.

Reply:
Oops, this was a mistake that has been reverted